build-external-api-jar/project/Dependencies.scala

```diff
@@ -0,0 +1,26 @@
+import sbt._
+
+object Dependencies {
+  val junit = Seq(
+    "junit" % "junit" % "4.12" % Test,
```

Comment:
Can we use Junit 4.13?

Reply:
Done here https://github.com/pantsbuild/intellij-pants-plugin/pull/599